index.html

```diff
@@ -1,17 +1,20 @@
+
 <!DOCTYPE html>
 <html>
    <head>
-      <link rel="stylesheet" type="text/css" href="style.css">
+      <link rel="stylesheet" type="text/css" href="contri.css">
```

Comment:
The name of the CSS file should be kept as "style.css" or you should rename the CSS file to contri